models/runs/grouping.mdx

```diff
@@ -49,6 +36,34 @@ To group runs by one or more columns:
 1. If you selected more than one column, drag them to change the grouping order.
 1. Click anywhere outside of the form to dismiss it.
 
+### 4. Manage groups in the UI
+
+#### Manage a run
+After a run is initialized, you can move it to a new group from your workspace or its **Runs** page.
+
+1. Navigate to your W&B project.
+1. Select the **Workspace** or **Runs** tab from the project sidebar.
+1. To move a single run, search or scroll to it. Hover over the run name, click the three vertical dots, then click **Move to another group**.
+1. To create a new group, click **New group**. Type a group name, then submit the form.
+1. To delete a run, click the three vertical dots, then click **Delete run**.
+1. To move a run, select the new group from the list, then click **Move**.
+
+#### Manage runs in bulk
+1. If necessary, [toggle grouping by columns](#toggle-grouping-by-column-in-the-ui), grouping by the **Group** column.
```

Comment:
Should we add an [intro sentence](https://developers.google.com/tech-writing/one/lists-and-tables) to set off these lists?